### DISCUSSION TITLE
Improve zoom feature on desktop

### DISCUSSION THREAD

**Discussion by ua741** [ente-io/ente#525] [Enhancements]
Reported by [candroid-man](https://github.com/candroid-man) [here ](https://github.com/ente-io/frame/issues/244)
> For desktop, when you click it zooms in like 10x and there is no way of controlling it, so maybe a slider would be a good idea

**Comment by iksnawias** [comment]
Or simply use the mouse scroll to zoom in/out, like in Gphotos ;)
